### ISSUE TITLE
New user message uses Lego within a paragraph

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#2755]
**[Submitted to the original trac issue database at 1.56am, Monday, 1st March 2010]**

```
<%= t 'user.account.new image' %> <span class="minorNote"><%= t 'user.account.image size hint' %>
```

This should be one message with two variables so languages can dictate in which order these two things appear.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 8.31am, Monday, 1st March 2010]**

I disagree - given the fact that one is formatted as a "minor note" in smaller type, and that the other needs to line with the other options I don't see how this could work.

The first one is the "option" that the user needs to choose from the menu of possible actions presented by the radio button and the other is a hint about what to upload if they choose that option.

In fact my original thought was to put the second one under the file upload field, completely separate from the first.

**Comment by tomhughes** [comment]
**Author: avarab@gmail.com**
**[Added to the original trac issue at 8.58am, Monday, 1st March 2010]**

Replying to [comment:1 TomH]:
> I disagree - given the fact that one is formatted as a "minor note" in smaller type, and that the other needs to line with the other options I don't see how this could work.
> 
> The first one is the "option" that the user needs to choose from the menu of possible actions presented by the radio button and the other is a hint about what to upload if they choose that option.
> 
> In fact my original thought was to put the second one under the file upload field, completely separate from the first.

Yes it needs to line up, that's the point.

```
image current the Keep
image current the Remove
image current the Replace (best work 100x100 least at images square)
```

Read that from left-to-right as an Arabic or Hebrew speaker would? Do you want the details about image size to come before information about what the option actually does?

**Comment by tomhughes** [comment]
**[Added to the original trac issue at 12.12pm, Monday, 1st March 2010]**

(In [20233]) Move image size hint to avoid issues with LTR languages. Closes #2755.
